examples/hypervisor_manager.tf

```diff
@@ -0,0 +1,48 @@
+provider "oneview" {
+	ov_username = "Administrator"
+	ov_password = "admin123"
+	ov_endpoint = "https://10.50.9.31/"
+	ov_sslverify = false
+	ov_apiversion = 1200
+	ov_ifmatch = "*"
+}
+
+data "oneview_scope" "scope_obj" {
+        name = "test_scope"
+}
+/*
+resource "oneview_hypervisor_manager" "HypervisorManager" {
```

Comment:
Comment either update or create requests, but not both, and write the description for these requests like "uncomment the following for update".

Reply:
Done